### ISSUE TITLE
Docker compose file has incorrect pahts on Windows environments

### ISSUE THREAD

**Issue by wblachowski** [Cognifide/gradle-aem-multi#71]
Hi!
I noticed that paths created in `docker-compose.yml` file are incorrect on Windows. They contain both backslashes and forward slashes. This leads to `docker compose` script failure due to "unknown escape character".

In my case the template from `docker-compose.yml.peb` file:
```
    volumes:
      - "{{ environment.dockerRootPath }}/distributions/mod_dispatcher.so:/usr/local/apache2/modules/mod_dispatcher.so"
      - "{{ environment.dockerRootPath }}/httpd/cache:/opt/aem/dispatcher/cache"
      - "{{ environment.dockerRootPath }}/httpd/logs:/usr/local/apache2/logs"
      - "{{ environment.dockerConfigPath }}/httpd/conf:/usr/local/apache2/conf"
```
Gets resolved to:
```
    volumes:
      - "C:\testwindows\aem\.environment/distributions/mod_dispatcher.so:/usr/local/apache2/modules/mod_dispatcher.so"
      - "C:\testwindows\aem\.environment/httpd/cache:/opt/aem/dispatcher/cache"
      - "C:\testwindows\aem\.environment/httpd/logs:/usr/local/apache2/logs"
      - "C:\testwindows\aem\gradle\environment/httpd/conf:/usr/local/apache2/conf"
```

**Comment by pun-ky** [comment]
Thanks for reporting it. 

Strange because I had tested this on Windows many times without such issues.

However the fix is rather easy. Just slashes need to be normalized to forward slashes or OS specific. 

Don't hesitate to implement a fix or... @mierzwid please handle it if I can ask you for that 😉

Greetings

**Comment by pun-ky** [comment]
BTW the issue should be rather reported in GAP repo because I suppose generated paths are wrong / it's GAP issue.

**Comment by pun-ky** [comment]
Are you using Docker Desktop? If yes, yep that might be an issue to be fixed. But... Then you could ommit variables and use relative paths to unblock you.

**Comment by pun-ky** [comment]
Or just use Docker Toolbox instead of Desktop until GAP with fix become released.

**Comment by wblachowski** [comment]
Thanks for quick response @pun-ky. I did a workaround of hardcoding correctly formated paths. I opened a [new issue](https://github.com/Cognifide/gradle-aem-plugin/issues/401) under GAP repo, so I guess this one can be closed.

**Comment by pun-ky** [comment]
Closing as fixed in 7.0.4, example updated.
